python/tests/misc.py

```diff
@@ -18,5 +20,91 @@ def test_cmake_files_exists(self):
         )
 
 
+class TestConverter(unittest.TestCase):
+    """
+    Tests the hyperparameter conversions in `rascaline.utils`
+    """
+
+    def test_mode(self):
+        from rascaline.utils import convert_old_hyperparameter_names
+
+        with self.assertRaises(ValueError):
+            convert_old_hyperparameter_names({}, mode="BadMode")
+
+    def test_errant_params(self):
+        with self.assertRaises(ValueError):
+            convert_old_hyperparameter_names({"bad_param": 0}, mode="librascal")
+        with self.assertRaises(ValueError):
+            convert_old_hyperparameter_names({"bad_param": 0}, mode="dscribe")
+
+    def test_not_gto(self):
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
that works, but I've been using `self.assertWarns` everywhere else, so it might be better to use it here as well for consistency